src/views/Home.jsx

```diff
@@ -19,6 +49,23 @@ export function Home({ data, setListPath }) {
 						/>
 					))}
 			</ul>
+
+			<form onSubmit={handleSubmit}>
+				<h3>Create New Shopping List</h3>
+				<label htmlFor="newListName">Name Your List</label>
+				<br />
+				<input
+					type="text"
+					value={inputValue}
+					onChange={handleChange}
+					name="newListName"
+					required
+					aria-label="Shopping List Name"
+					aria-required="true" // Indicates that this field is required
```

Comment:
praise: I didn't know this aria-label existed, thanks for teaching me something new!